go/mysql/replication_status.go

```diff
@@ -41,8 +41,22 @@ type ReplicationStatus struct {
 	MasterPort           int
 	MasterConnectRetry   int
 	MasterUUID           SID
+	IOThreadState        IOThreadState
```

Comment:
Thinking about this further..
Why not remove the two bool fields, add a `SQLThreadState` and a wrapper func `IOThreadRunning` that returns a bool? Maybe a similar wrapper func `SQLThreadRunning` as well.

Reply:
That's a good idea. I looked up both MySQL and MariaDB docs and as far as I can tell, SQL_Thread_Running only returns "Yes", or "No", so I think we can keep that as a `bool`. Implemented the rest of your suggestion.